src/main/java/tech/jhipster/controlcenter/config/gateway/SetAuthorizationHeaderGatewayFilterFactory.java

```diff
@@ -0,0 +1,40 @@
+package tech.jhipster.controlcenter.config.gateway;
+
+import static tech.jhipster.controlcenter.security.jwt.JWTFilter.AUTHORIZATION_HEADER;
+
+import org.springframework.cloud.gateway.filter.GatewayFilter;
+import org.springframework.cloud.gateway.filter.factory.AbstractGatewayFilterFactory;
+import org.springframework.http.server.reactive.ServerHttpRequest;
+import org.springframework.stereotype.Component;
+import org.springframework.util.StringUtils;
+import tech.jhipster.controlcenter.security.jwt.TokenProvider;
+
+@Component
+public class SetAuthorizationHeaderGatewayFilterFactory extends AbstractGatewayFilterFactory<Object> {
```

Comment:
is it possible to add tests for this class ?

Reply:
I think it is possible, I will try to add tests for this class.

Reply:
I added tests for this class in the next commit :+1: